src/Enum/NotificationType.php

```diff
@@ -20,4 +20,7 @@ class NotificationType extends Enum
 
     /** @var string */
     protected const PULL_REQUEST = 'PullRequest';
+
+    /** @var string  */
+    protected const REPOSITORY_VULNERABILITY_ALERT = 'RepositoryVulnerabilityAlert';
```

Comment:
You've forgot `@method static NotificationType REPOSITORY_VULNERABILITY_ALERT()` on the PHPDoc